src/asmcnc/apps/drywall_cutter_app/screen_drywall_cutter.py

```diff
@@ -247,6 +249,9 @@ def on_pre_enter(self):
         self.kb.set_numeric_pos((scaling_utils.get_scaled_width(565), scaling_utils.get_scaled_height(115)))
         self.drywall_shape_display_widget.check_datum_and_extents()  # update machine value labels
 
+        if not self.model_manager.is_machine_drywall():
+            self.drywall_shape_display_widget.canvas_image.source = "./asmcnc/apps/drywall_cutter_app/img/canvas_with_logo_shapes.png"
+
```

Comment:
Can you move the original source call from the builder string into this if-else? So all the image loading is in one place.